core/src/main/java/org/apache/iceberg/util/ContentFileUtil.java

```diff
@@ -84,4 +85,17 @@ public static String referencedDataFileLocation(DeleteFile deleteFile) {
     CharSequence location = referencedDataFile(deleteFile);
     return location != null ? location.toString() : null;
   }
+
+  public static boolean isDV(DeleteFile deleteFile) {
```

Comment:
same as I mentioned on the other PR. This isn't probably needed anymore, since it exists in `ScanTaskUtil`

Reply:
This is actually intentional. I originally wanted to move `ContentFileUtil` to `api` but then we decided against that, as it required moving more classes and increasing the API surface. Therefore, we added a simple utility for what was needed in `api`. I do want `ContentFileUtil` to contain all the logic for content files, it is just a matter of one duplicate method.